indico/modules/events/contributions/controllers/management.py

```diff
@@ -679,3 +680,10 @@ def _process_POST(self):
         self.subcontrib.references.append(reference)
         db.session.flush()
         return self.jsonify_reference(reference)
+
+
+class RHCloneContribution(RHManageContributionBase):
+    def _process(self):
+        clone_contribution(self.event, self.contrib, False)
+        db.session.commit()
+        return jsonify_data(html=self.list_generator.render_list()['html'])
```

Comment:
What about using the same way like we do for all the other endpoints?

    return jsonify_data(**self.list_generator.render_list())

Not sure if there's anything besides `html` but if it's used to update the contribution list it'd probably break if we ever added something else since here that would then be missing.